Files.TOOD/un_034050.csv

```diff
@@ -0,0 +1,26 @@
+name01#0,"""? ? ?"""
+name02#0,"""? ? ?"""
+un_034050_0010#0,"""<%me>! Over here! I think we lost her!"""
+un_034050_0020#0,"""...Phew. We're safe for now. Hey, Echo, it's..."""
+un_034050_0030#0,"""Ze-Zeno, Zeno! Up, Look up!"""
```

Comment:
our spell checker script think `Ze` is wrong
You may want to edit `_aspell/PSO2.en_Chat.dict` to add Ze to the word list

Reply:
Do I have permission to do that?